builder/processor-tools/src/main/java/io/helidon/builder/processor/tools/DefaultBuilderCreatorProvider.java

```diff
@@ -1746,12 +1822,17 @@ private String normalize(String name) {
         return name.equals("?") ? "Object" : name;
     }
 
-    private String quotedTupleOf(String key,
+    private String quotedTupleOf(TypeName valType,
+                                 String key,
                                  String val) {
-        assert (Objects.nonNull(key));
-        assert (BuilderTypeTools.hasNonBlankValue(val)) : key;
-        if (key.equals("value") && ConfiguredOption.UNCONFIGURED.equals(val)) {
-            val = ConfiguredOption.class.getName() + ".UNCONFIGURED";
+        assert (key != null);
+        assert (hasNonBlankValue(val)) : key;
+        boolean isEnumLikeType = isEnumLikeType(valType, key, val);
+        if (isEnumLikeType) {
+            val = valType + "." + val;
+        } else if (key.equals("value") && val.startsWith(ConfiguredOption.class.getName())) {
+            // NOP; process this as-is
+            int debugMe = 0;
```

Comment:
Is this intentionally left here?

Reply:
yes, for debugging purposes.

Reply:
Instead of simulating nop instruction, please use conditional breakpoint, or logger with debug level

Reply:
changed to use `assert (true); // for debug breakpoints only`